From 7b30ea0a2d70b2e17b929f8af0eefbc54016a42c Mon Sep 17 00:00:00 2001
From: Fabian Greffrath <fabian@greffrath.com>
Date: Thu, 27 Aug 2020 08:06:21 +0200
Subject: [PATCH] consider Sigil's mixed case file names

We already try some common cases when trying to find and load
a WAD file, but never the mixed case that the Sigil WADs come
with by default.

Hopefully fixes #655, thanks @kitchen-ace and @kbterkelsen.
---
 src/doom/d_pwad.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/doom/d_pwad.c b/src/doom/d_pwad.c
index e0be6a22..8a659c77 100644
--- a/src/doom/d_pwad.c
+++ b/src/doom/d_pwad.c
@@ -36,9 +36,9 @@ void D_LoadSigilWad (void)
 	char *dirname;
 
 	const char *const sigil_wads[] = {
-		"SIGIL_V1_21.WAD",
-		"SIGIL_V1_2.WAD",
-		"SIGIL.WAD"
+		"SIGIL_v1_21.wad",
+		"SIGIL_v1_2.wad",
+		"SIGIL.wad"
 	};
 
 	struct {
-- 
2.28.0

